### PR TITLE
solution for exercise 2

### PR DESCRIPTION
**Pull request by christianrietzsch**

### CODE REVIEW

File: Exercises/ChristianRietzsch/exercises_02.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+                reg2 = memory[memory[current+2]]
+                reg3 = memory[current+3]
+                if(element == 1):
+                    memory[reg3] = reg1 + reg2
+                else:
+                    memory[reg3] = reg1 * reg2
+            case 99:
+                return memory[0];
+            case _:
+                print("Error, Illegal Instruction: {memory[current]}")
```
Comment (DrKeks1010) [inline Exercises/ChristianRietzsch/exercises_02.py:48]: This should be an f string, i suppose

**Review by DrKeks1010 — APPROVED** [review]
Overall a great solution.

Be will find out tomorrow, if singel digits like '2' should also be interpreted as strings with just one character or if the program should exit after parsing '2' to a number, so you might need to adjust this.

File: Exercises/ChristianRietzsch/exercises_02.py (hunk 1 above)
Comment (GeFritz) [reply Exercises/ChristianRietzsch/exercises_02.py:48]: bro lass mal paar reviews übrig

**Review by fabischw — CHANGES_REQUESTED** [review]
Task 2 does not accept the input "1" as both char and number. It only adds the "1" to the number list, not to the char list.

**Review by fabischw — APPROVED** [review]
problem has been solved and requested changes implemented. well done.

**Pull request merged**